dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/sms/SmsMessageSenderTest.java

```diff
@@ -72,13 +75,16 @@
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
 
 import com.google.common.collect.Sets;
 
 /**
  * @author Zubair Asghar.
  */
 @ExtendWith( MockitoExtension.class )
+@MockitoSettings( strictness = Strictness.LENIENT )
```

Comment:
Please adapt your tests so that we use mockito strict. If you see tests with `@MockitoSettings( strictness = Strictness.LENIENT )` its because previously that was the default mockito setting. When migrating Mockito/JUnit 5 we had to add this setting since a lot of the tests would fail. We should make an effort to make all our tests use the now default mockito strict mode. This helps keep tests easier to read/maintain going forward.